### PR TITLE
chore(SCSS): created skeleton for testing SCSS

### LINKED ISSUE #952: SCSS: Add unit tests
https://seesparkbox.com/foundry/how_and_why_we_unit_test_our_sass

- [x] Add new tool as a dependency
- [x] Create a `.spec.scss` file by each helper SCSS file
- [x] Add tests for at least one mixin and one function
- [ ] Create a new command to run the tests in `package.json`
- [x] Make sure that when the CI runs all our tests, these new tests are also run

NOTE: This ticket doesn't need all items tested. Just a couple to verify the approach works. We can then open up tickets to test other functions etc. (Some of the shadow functions are fairly complex and could blow up the scope of this ticket)

### PR DESCRIPTION
**Pull request by shani-terminus**
ISSUES CLOSED: #952

### CODE REVIEW

**Comment by shani-terminus** [comment]
"testMatch" is the only change in the jest.config. the rest is conversion of `"` to `'`

File: terminus-ui/scss/helpers/test-sass.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef */
```
Comment (benjamincharity) [inline terminus-ui/scss/helpers/test-sass.js]: What is the undefined item we are ignoring?

Comment (shani-terminus) [reply terminus-ui/scss/helpers/test-sass.js]: The True mixins: `describe` and `test`. I can probably make two one-liners, but it was useful having the whole doc when in discovery mode.

Comment (benjamincharity) [reply terminus-ui/scss/helpers/test-sass.js]: Hmm I wonder why this is needed but isn't needed in all the other files that use those two..

Comment (benjamincharity) [reply terminus-ui/scss/helpers/test-sass.js]: Alright, figured out why this was happening. Our ESLint configuration sets the `jest` environment for files ending with `spec` or `mock` so it wasn't catching this new file naming convention. We can remove this ESLint comment by extending the overrides section of our ESLint configuration:

### `.eslintrc.js`

```javascript
module.exports = {
  "extends": ["@terminus/eslint-config-frontend/development"],
  "parserOptions": {
    "ecmaVersion": 6,
    "project": "./tsconfig.json",
    "sourceType": "module"
  },
  "rules": {
    "no-console": [
      "error",
      {
        "allow": [
          "warn"
        ]
      }
    ]
  },
  'overrides': [
    {
      'files': [
        '**/*.spec.ts',
        '**/*.mock.ts',
        '**/test-sass.js',
      ],
      'env': {
        'jest': true,
      },
    },
  ],
}
```

### `.eslintrc.ci.js`

```javascript
module.exports = {
  "extends": ["@terminus/eslint-config-frontend"],
  "parserOptions": {
    "ecmaVersion": 6,
    "project": "./tsconfig.json",
    "sourceType": "module"
  },
  "rules": {
    "no-console": [
      "error",
      {
        "allow": [
          "warn"
        ]
      }
    ]
  },
  'overrides': [
    {
      'files': [
        '**/*.spec.ts',
        '**/*.mock.ts',
        '**/test-sass.js',
      ],
      'env': {
        'jest': true,
      },
    },
  ],
}
```

Comment (benjamincharity) [inline terminus-ui/scss/helpers/test-sass.js]: Let's make the configuration change so we can remove this comment.

**Review by benjamincharity — CHANGES_REQUESTED** [review]

**Review by benjamincharity — APPROVED** [review]

File: terminus-ui/scss/helpers/_breakpoints.spec.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        @include assert {
+      @include output {
+        @include bp(layout-xs);
+      }
+      // is there a way to tests @media queries?
+      @include expect {
+
+      }
+    }
+  }
```
Comment (atlwendy) [inline terminus-ui/scss/helpers/_breakpoints.spec.scss]: Is this commented out block for future reference?

**Review by atlwendy — APPROVED** [review]
👍 scss spec is new to me.

**Pull request merged**